packages/network-controller/src/create-auto-managed-network-client.test.ts

```diff
@@ -95,98 +98,219 @@ describe('createAutoManagedNetworkClient', () => {
         expect('request' in provider).toBe(true);
       });
 
-      it('returns a provider proxy that acts like a provider, forwarding requests to the network', async () => {
-        mockNetwork({
-          networkClientConfiguration,
-          mocks: [
-            {
-              request: {
-                method: 'test_method',
-                params: [],
-              },
-              response: {
-                result: 'test response',
+      describe('when accessing the provider proxy', () => {
```

Comment:
The diff is bad here but I ended up grouping the existing tests under a new `describe`.